### DISCUSSION TITLE
[question] StoreKit: alternative to recepit validation?

### DISCUSSION THREAD

**Discussion by Aiosa** [hyochan/react-native-iap#3042] [Polls]
Sorry if this is offtopic here, I just could not find any community forum link and here it only offers a bug report template. So I want to ask a question, not to report a bug.

I was using `validateReceiptIos` return value to obtain certain things such as the expiration date.
Now with SK2 there are no receipts. Is there a supported way of obtaining the transaction-related metadata similar to how the response of receipt validation worked? I could not find any new transaction-related getter-like methods in the api. And although I am suspicious that `verificationResultIOS` could do this for me, I just dont know how to read the data from it (seems like base64-encoded `ES256`).

Thanks!

**Comment by mathysFor** [comment]
hello, did you find a solution ?
